boards/riscv/icev_wireless/icev_wireless.dts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2022 Friedt Professional Engineering Services, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3.dtsi>
+#include "icev_wireless-pinctrl.dtsi"
+
+/ {
+	model = "icev_wireless";
+	compatible = "espressif,esp32c3";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &usb_serial;
+		zephyr,shell-uart = &usb_serial;
+		zephyr,flash = &flash0;
+	};
+
+	aliases {
+		led0 = &led_green;
+		i2c-0 = &i2c0;
+		sw0 = &user_button1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_green: led_0 {
+			gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
+			label = "LED Green";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button1: button_1 {
+			label = "User SW1";
```

Comment:
Should this be labeled "Boot"? Just trying to match this with the picture of the board.

Reply:
I actually don't see that as a repeating theme in Zephyr and see "User SW1" as more of a repeating theme. Can you point out a specific example? Note: after boot, the button may be used for user purposes".

Reply:
Yeah the Nordic boards somewhat match the node name with what's on the board silkscreen (Button 1... LED1...), same for few STM32 boards I have kicking around (User LD3/LD4 on the f429-disc1). Probably not super important but here I would expect a button labeled "SW1" on the board (which is the refdes of the reset button on that board btw, the one labeled "Boot" is SW2).

Reply:
![x](https://user-images.githubusercontent.com/891546/180884852-8a361863-5947-4c68-aa58-e13c64cf52af.png)
This is what I'm taking as reference (and the board picture)

Reply:
Pedantically speaking, it should probably be labeled SW2.

https://github.com/ICE-V-Wireless/ICE-V-Wireless/blob/main/docs/esp32c3_fpga_schematic.pdf

Reply:
Yeah that's what I would name it if I was picking a refdes, otherwise "Boot" to match the silkscreen.

Reply:
In terms of Zephyr convention:
```shell
% grep -R "label.*=.*SW1" $(find boards -name '*.dts')   
boards/arm/ubx_evkninab3_nrf52840/ubx_evkninab3_nrf52840.dts:                   label = "Push button switch 0 (SW1)";
boards/arm/warp7_m4/warp7_m4.dts:                       label = "User SW1";
boards/arm/frdm_kl25z/frdm_kl25z.dts:                   label = "User SW1";
boards/arm/bt610/bt610.dts:                     label = "Boot button switch 1 (SW1)";
boards/arm/lora_e5_dev_board/lora_e5_dev_board.dts:                     label = "SW1";
boards/arm/pan1770_evb/pan1770_evb.dts:                 label = "SW1 on EVB";
boards/arm/lpcxpresso54114/lpcxpresso54114_m4.dts:                      label = "User SW1";
boards/arm/usb_kw24d512/usb_kw24d512.dts:                       label = "User SW1";
boards/arm/bt510/bt510.dts:                     label = "Push button switch 1 (SW1)";
boards/arm/ubx_evkninab1_nrf52832/ubx_evkninab1_nrf52832.dts:                   label = "Push button switch SW1";
boards/arm/bl653_dvk/bl653_dvk.dts:                     label = "Push button switch 1 (SW1)";
boards/arm/bl653_dvk/bl653_dvk.dts:                     label = "Push button switch 4 (SW10)";
boards/arm/colibri_imx7d_m4/colibri_imx7d_m4.dts:                       label = "User SW1";
boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832.dts:                   label = "Push button SW1";
boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts:                        label = "User SW1";
boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts:                    label = "User SW1";
boards/arm/lpcxpresso55s69/lpcxpresso55s69_ns.dts:                      label = "User SW1";
boards/arm/pan1781_evb/pan1781_evb.dts:                 label = "SW1 on EVB";
boards/arm/nucleo_wb55rg/nucleo_wb55rg.dts:                     label = "SW1";
boards/arm/pan1780_evb/pan1780_evb.dts:                 label = "SW1 on EVB";
boards/arm/bl654_dvk/bl654_dvk.dts:                     label = "Push button switch 1 (SW1)";
boards/arm/bl654_dvk/bl654_dvk.dts:                     label = "Push button switch 4 (SW10)";
boards/arm/pan1782_evb/pan1782_evb.dts:                 label = "SW1 on EVB";
boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts:                        label = "User SW1";
boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts:                        label = "SW1_BUCK_REG";
boards/arm/bl654_sensor_board/bl654_sensor_board.dts:                   label = "Push button switch 1 (SW1)";
boards/arm/ubx_evkannab1_nrf52832/ubx_evkannab1_nrf52832.dts:                   label = "Push button switch SW1";
boards/arm/nucleo_wl55jc/nucleo_wl55jc.dts:                     label = "SW1";
boards/arm/lpcxpresso55s28/lpcxpresso55s28.dts:                 label = "User SW1";
boards/riscv/esp32c3_devkitm/esp32c3_devkitm.dts:                       label = "User SW1";
boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts:                 label = "User SW1";
```

Also, if you would like to see the exact same convention on a directly related board
<img width="718" alt="Screen Shot 2022-07-25 at 6 34 54 PM" src="https://user-images.githubusercontent.com/983494/180885591-352e50d6-0671-4322-8da9-b14f6c4d279d.png">

It might not be exactly what you expect, but it's conventional :-)

Reply:
Hey I hope at least some of those are actually SW1 somewhere! :-) Aaawlright let's go with it.